docs/design/storage-node-windows-installer.md

```diff
@@ -13,73 +13,67 @@ Similarly, Docker ends up using more resources than running natively on Windows.
 Therefore, it would be nice to be able to run Storage Node without Docker.
 
 For an easy setup process we would need an installer that:
-1. Sets up Storage Node to run in background,
-2. sets up automatic updates and
-3. ensures we log everything properly.
+1. Sets up Storage Node to run in background.
+1. Sets up automatic updates.
+1. Ensures we log everything properly.
 
 We need to take care that we:
-* avoid triggering UAC unnecessarily,
-* start storage node before logging in and
-* restart on crashes.
+* Avoid triggering UAC unnecessarily.
+* Start storage node before user login.
+* Restart on crashes.
```

Comment:
crash of application or OS or both?

Reply:
Crash of the application. I am not sure if it is possible for Windows Service Manager to detect OS crashes.